### PR TITLE
Added description/examples to queryDeleteColumn

### PR DESCRIPTION
**Pull request by danacoder**

### CODE REVIEW

File: data/en/querydeletecolumn.json [hunk 1]
```diff
@@ -5,14 +5,21 @@
 	"member": "query.deleteColumn(column)",
 	"returns": "array",
 	"related": [],
-	"description": "",
+	"description": "Deletes a column within a query object.",
 	"params": [
-		{"name": "query", "description": "", "required": true, "default": "", "type": "query", "values": []},
-		{"name": "column", "description": "", "required": true, "default": "", "type": "string", "values": []}
+		{"name": "query", "description": "The query to modify.", "required": true, "default": "", "type": "query", "values": []},
+		{"name": "column", "description": "The name of the column to delete.", "required": true, "default": "", "type": "string", "values": []}
```
Comment (shaedrich) [inline data/en/querydeletecolumn.json]: If you omit the "the" in these two param descriptions, it'll look fine ;-)

**Comment by danacoder** [comment]
Thanks again for the style suggestion.  I've made the change.  Also, submitted a second request to tweak the queryDeleteRow that had a stray "the" and some redundant text.

**Comment by shaedrich** [comment]
Nice :+1:

Would be better to have it all in one, but I guess @pfreitag won't have a problem with approving this alongside with #820

**Comment by pfreitag** [comment]
Thanks @danacoder

**Pull request merged**